src/codeu/chat/server/AuthorityModel.java

```diff
@@ -0,0 +1,13 @@
+public interface AuthorityModel
```

Comment:
This interface has compile errors because a few things are missing: imports, and the package declaration. Can you please add them?